tests/contrib/django_util/test_views.py

```diff
@@ -197,14 +199,15 @@ def test_callback_handles_bad_flow_exchange(self, pickle):
             state=json.dumps(self.fake_state),
             redirect_uri=request.build_absolute_uri('oauth2/oauth2callback'))
 
-        session_key = 'google_oauth2_flow_{0}'.format(self.CSRF_TOKEN)
-        self.session[session_key] = pickle.dumps(flow)
+
+        session_key ='google_oauth2_flow_{0}'.format(self.CSRF_TOKEN)
```

Comment:
I was wondering why this showed up in the diff, then I noticed you're missing a space after the `=` sign